### PR TITLE
QF-2059-SendResponseNotificationToEmployers

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

File: src/SFA.DAS.RequestApprenticeTraining.Api/Controllers/EmployerRequestController.cs [hunk 1]
```diff
@@ -198,10 +199,12 @@ public async Task<IActionResult> SubmitProviderResponse(long ukprn, SubmitProvid
             try
             {
                 var response = await _mediator.Send(new SubmitProviderResponseCommand
-                { 
+                {
                     Ukprn = ukprn,
+                    ContactName = parameters.ContactName,
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Api/Controllers/EmployerRequestController.cs]: Your branch is not sync as I've seen these changes somewhere before?

File: src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetEmployerRequestsForResponseNotification/GetEmployerRequestsForResponseNotificationQueryHandler.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        {
+
+            var requests = await _employerRequestEntityContext.GetForResponseNotification();
+
+            return new GetEmployerRequestsForResponseNotificationQueryResult
+            {
+                EmployerRequests =
+                    requests.Select(entity => (EmployerRequestForResponseNotification)entity).ToList()
+            };
+
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetEmployerRequestsForResponseNotification/GetEmployerRequestsForResponseNotificationQueryHandler.cs]: White space city?

File: src/SFA.DAS.RequestApprenticeTraining.Domain/Models/SubmitProviderResponseParameters.cs [hunk 3]
```diff
@@ -9,5 +9,9 @@ public class SubmitProviderResponseParameters
         public string Email { get; set; }
         public string Phone { get; set; }
         public string Website { get; set; }
+        public string ContactName { get; set; }
+        public Guid RespondedBy { get; set; }
+
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Domain/Models/SubmitProviderResponseParameters.cs]: White space

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RequestApprenticeTraining.Api/Controllers/EmployerRequestController.cs (hunk 1 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Api/Controllers/EmployerRequestController.cs]: merged master -> 2037 -> 2059 and it should be in sync now

File: src/SFA.DAS.RequestApprenticeTraining.Domain/Models/SubmitProviderResponseParameters.cs (hunk 3 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Domain/Models/SubmitProviderResponseParameters.cs]: removed

File: src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetEmployerRequestsForResponseNotification/GetEmployerRequestsForResponseNotificationQueryHandler.cs (hunk 2 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetEmployerRequestsForResponseNotification/GetEmployerRequestsForResponseNotificationQueryHandler.cs]: removed

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**